3DAmsterdam/Assets/Stadsmodel/Buildings/BuildingTileManager.cs

```diff
@@ -1,372 +1,364 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.Networking;
+﻿using BruTile;
 using ConvertCoordinates;
-using BruTile;
 using System;
+using System.Collections;
+using System.Collections.Generic;
 using System.Text.RegularExpressions;
+using UnityEngine;
+using UnityEngine.Networking;
 
-/// <summary>
-/// enum to indentify the status of a BuildingTile
-/// </summary>
-public enum BuildingTileStatus
-{
-    WaitingForDownload,     //ready to be downloaded
-    Downloading,            //download is active
-    PendingBuild,             //waiting for display
-    Building,             //getting displayed
-    Built,              //Assetbundlecontent is displayed
-    PendingDestroy
-}
-class BuildingTileData
+
+class TileData
 {
-    public Vector3 id;
-    public List<GameObject> Gameobjecten = new List<GameObject>();
-    public AssetBundle AB;
-    public bool TeVerwijderen = false;
-    public BuildingTileStatus Status;
+    public Vector3 tileID;
+    public List<GameObject> gameObjects = new List<GameObject>();
+    public AssetBundle assetBundle;
+
+    public TileStatus Status;
     
-    public BuildingTileData(Vector3 TileID)
+    public TileData(Vector3 TileID)
     {
-        id = TileID;
-        Status = BuildingTileStatus.WaitingForDownload;
+        tileID = TileID;
     }
 }
 
-
-
 public class BuildingTileManager : MonoBehaviour
 {
     
 
-    public string BuildingURL = "file:///D://Git/A3d/AssetBundles/WebGL/";
-    public Material GebouwMateriaal;
-    public Material HighLightMateriaal;
-    public float Max_Afstand_Daken = 500;
-    public float Max_Afstand_BAG = 1000;
-    public float Max_Afstand_Top10 = 3000;
-    private CameraView CV;
-    private Extent vorigeCV = new Extent(0, 0, 0, 0);
-    public int MAX_Concurrent_Downloads = 5;
-    private Boolean BijwerkenGereed = true;
-
-    private Dictionary<float, string> gebouwenlijst = new Dictionary<float, string>();
-    private Dictionary<Vector3, BuildingTileData> buildingTiles = new Dictionary<Vector3, BuildingTileData>();
+    public string assetURL = "https://3d.amsterdam.nl/web/AssetBundles/Gebouwen/";
+    public Material defaultMaterial;
+    public Material highLightMaterial;
+    public float maximumDistanceLOD2 = 500;
+    public float maximumDistanceLOD1 = 1000;
+    public float maximumDistanceLOD0 = 3000;
+    private CameraView cameraViewExtent;
+    private Extent previousCameraViewExtent = new Extent(0, 0, 0, 0);
+    public int maximumConcurrentDownloads = 5;
+
+    private bool tileUpdateCompleted = true;
+
+    private Dictionary<Vector3, TileData> activeTileList = new Dictionary<Vector3, TileData>();
     private List<Vector3> PendingDownloads = new List<Vector3>();
```

Comment:
@mvangog Ik zie dat je PascalCase heb gebruikt voor de list namen, maar dat zijn in principe gewoon variabelen die ook camelCase mogen. Dan zijn ze ook niet te verwarren met methodes zodra iemand ze voorbij ziet komen in wat regels code verderop.